src/file.cpp

```diff
@@ -2192,57 +2116,6 @@ typedef struct _FILE_ALLOCATED_RANGE_BUFFER {
 #endif
 	}
 
-	boost::int64_t file::sparse_end(boost::int64_t start) const
```

Comment:
Just to confirm: Do you want me to restore the function? I rather remove it and bring it back with some unit tests if it ever needed, but you tell me.